Helper/RunnerManager.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Liip\MonitorBundle\Helper;
+
+use Liip\MonitorBundle\Runner;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class RunnerManager
+{
+    /** @var ContainerInterface */
+    private $container;
+
+    /**
+     * @param ContainerInterface $container
+     */
+    public function __construct(ContainerInterface $container)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * @param string $group
+     *
+     * @return null|Runner
+     */
+    public function getRunner($group)
```

Comment:
so my last open question is if we should allow ``null`` here as a parameter and in that case use the default group